typescript/infra/scripts/interchain/deploy.ts

```diff
@@ -31,6 +31,7 @@ async function main() {
     multiProvider,
     configMap,
     core,
+    'ica2',
```

Comment:
What does this represent?

Reply:
a salt that a router deployment gives to the CREATE2 factory